server/src/main/scala/drt/server/feeds/stn/STNForecastXLSExtractor.scala

```diff
@@ -37,38 +37,36 @@ object STNForecastXLSExtractor {
 
     log.info(s"Extracting STN forecast flights from XLS Workbook located at $xlsFilePath")
 
-    val formatter = DateTimeFormat.forPattern("dd/MM/yyyy HH:mm")
-
     val lgwWorkSheet = workbook(xlsFilePath)
 
-    val sheet = sheetMapByIndex(0, lgwWorkSheet)
-
+    val sheet = sheetMapByIndex(3, lgwWorkSheet)
     val arrivalRows: Seq[STNForecastFlightRow] = for {
-      rowNumber <- 1 to sheet.getLastRowNum
+      rowNumber <- 2 to sheet.getLastRowNum
       row = sheet.getRow(rowNumber)
-      if row.getCell(0) != null
+      if row.getCell(1) != null && row.getCell(1).getCellType != Cell.CELL_TYPE_BLANK
     } yield {
-      val terminalCell = numericCell(0, row).map(t => s"T${t.toInt}")
-      val scheduledCell = stringCell(1, row).getOrElse("")
-      val flightNumberCell = stringCell(2, row)
-      val airportCell = stringCell(3, row)
-      val internationalDomesticCell = stringCell(4, row)
-      val totalCell = numericCell(5, row).getOrElse(0.0)
-      val transferPaxCell = numericCell(6, row).getOrElse(0.0)
-      val scheduled = SDate(DateTime.parse(scheduledCell, formatter).getMillis, DateTimeZone.forID(europeLondonId))
+      val scheduledCell = numericCell(1, row).getOrElse(0.0)
+      val airline = stringCell(2, row).getOrElse("")
```

Comment:
`airline` should be `carrierCode` if it's the airline's IATA / ICAO 2/3 letter code

Reply:
Sorry, worth renaming this `airline` to `carrierCode` just to maintain consistency throughout the app I think

Reply:
FR
U2
TK

are values in excel under airline column and column is just before flight Number . So I kept airline for extraction and also amended it to FlightNumber while creating flightCode